src/helpers/_spacing.scss

```diff
@@ -58,40 +100,53 @@
 
 /// Responsive margin
 ///
-/// Adds responsive margin using a 'spacing map', which defines different
-/// spacing values at different breakpoints. Wrapper for the
-/// `govuk-responsive-spacing` mixin.
+/// Adds responsive margin by fetching a 'spacing map' from the responsive
+/// spacing scale, which defines different spacing values at different
+/// breakpoints. Wrapper for the `govuk-responsive-spacing` mixin.
 ///
 /// @see {mixin} govuk-responsive-spacing
 ///
-/// @param {Map} $scale-map - Map of breakpoints and spacing values
+/// @param {Number} $responsive-spacing-point - Point on the responsive spacing
+/// scale, corresponds to a map of breakpoints and spacing values
 /// @param {String} $direction [all] - Direction to add spacing to
 ///   (`top`, `right`, `bottom`, `left`, `all`)
 /// @param {Boolean} $important [false] - Whether to mark as `!important`
 /// @param {Number} $adjustment [false] - Offset to adjust spacing by
 ///
+/// @example scss
+///   .element {
+///      @include govuk-responsive-margin(6, "left", $adjustment: 1px);
+///   }
+///
 /// @access public
 
-@mixin govuk-responsive-margin($scale-map, $direction: "all", $important: false, $adjustment: false) {
-  @include govuk-responsive-spacing($scale-map, "margin", $direction, $important, $adjustment);
+@mixin govuk-responsive-margin($responsive-spacing-point, $direction: "all", $important: false, $adjustment: false) {
+  @include govuk-responsive-spacing($responsive-spacing-point, "margin", $direction, $important, $adjustment);
```

Comment:
We could pass all arguments [like we do for mq](https://github.com/alphagov/govuk-frontend/blob/master/src/helpers/_media-queries.scss#L91-L95) if we wanted to avoid having to maintain the same signature in three places (though we'd need to update the docs anyway I guess…)

Reply:
Thank @36degrees. As discussed IRL, we need to pass the property (`margin` / `padding`) through to `_ govuk-responsive-spacing` so `$args...` wouldn't work in this scenario.